### PR TITLE
Change android play store description copy to match ios

### PR DESCRIPTION
**Pull request by nityas**
### Description

iOS store copy submitted deviated slightly from android current copy to get approved. This PR updates the Android Play Store description to match iOS App Store.

### Tested

copy change- proofread

### Other changes

n/a

### Backwards compatibility

Copy Change

### CODE REVIEW

**Comment by nityas** [comment]
Pending App Store Approval -- DO NOT MERGE

**Comment by martinvol** [comment]
As the e2e tests are failing, Do you @jeanregisser o @nityas tested that the app starts up with these changes?

**Comment by jeanregisser** [comment]
Yes the app was working.

**Review by jmrossy — APPROVED** [review]

**Pull request merged**
